docs/base/modelagem_bpmn.md

```diff
@@ -0,0 +1,47 @@
+# Modelagem BPMN
+
+## Versionamento
+| Versão | Data | Modificação | Autor |
+|--|--|--|--|
+|1.0| 27/01/2022| Criação do documento de modelagem BPMN | João Pedro Moura |
+|1.1| 28/01/2022| Adição dos fluxogramas de modelagem BPMN | João Pedro Moura |
+
+## Introdução
+<p align="justify">&emsp;&emsp; O Business Process and Notation (BPMN) ou notação de modelagem de processos de negócio, é um tipo de modelagem que através de um diagrama visa planejar, formalizar, modelar e melhorar a eficiência [1] dos diversos fluxos presentes no desenvolvimento de um software. Para isso, essa diagramação faz o uso da linguagem BPMN, padronizada e universal onde é possível destacar os seguintes elementos presentes dentro desse fluxograma: <b>Objetos de fluxo</b>, <b>Objetos de conexão</b>, <b>Raias</b> e <b>Artefatos</b>.</p>
```

Comment:
A definição do acrônimo BPMN está incorreta, é Business Process Model and Notation.